pkgs/applications/virtualization/ovmfvartool/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchFromGitHub, python3 }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "ovmfvartool";
+  version = "2021.06.16";
```

Comment:
Date separator should be "-" instead of ".".

Reply:
Also should have "unstable-".

Reply:
Remember to update commit message with new version.

Reply:
I did

Reply:
Looks good. Waiting CI.